### PR TITLE
Update MacOS uninstall documentation

### PR DESCRIPTION
**Pull request by mrpau-eduard**
## Summary

This will update the macOS uninstall documentation. This include two option how to uninstall the KA Lite in your macOS via `KA Lite Preferences`  or `KA-Lite_Uninstall.tool`

## Issues addressed
https://github.com/learningequality/ka-lite-installers/issues/446
#5440

### CODE REVIEW

File: docs/installguide/uninstall.rst [hunk 1]
```diff
@@ -11,14 +11,22 @@ _______
...
+1. Launch ``KA-Lite`` from your ``Applications`` folder.
 2. Click on the app icon at the menu bar.
 3. Click on ``Preferences`` in the menu option.
-4. Click the ``Reset App`` from the ``Advanced`` tab.
-5. You will be prompted that "This will reset app. Are you sure?", just click on ``OK`` button.
-6. Another dialog will appear asking your ``Password``, type your password then click on ``Ok`` button.
-7. Quit the ``KA-Lite Monitor`` app (do not click the ``Apply`` button!).
-8. Move the ``KA-Lite Monitor`` app to ``Trash``.
+4. Click the ``Uninstall KA Lite`` from the ``Preferences`` tab.
+5. if you want to delete your KA Lite data folder checked the ``Delete KA Lite data folder`` checkbox.
```
Comment (benjaoming) [inline docs/installguide/uninstall.rst]: Capital "If" :)

File: docs/installguide/uninstall.rst [hunk 2]
```diff
@@ -11,14 +11,22 @@ _______
...
+6. You will be prompted that ``Are you sure that you want to uninstall the KA Lite application?``, just click on ``OK`` button.
+7. Another dialog will appear asking your ``Password``, type your password then click on ``Ok`` button.
```
Comment (benjaoming) [inline docs/installguide/uninstall.rst]: No 6 and 7 can be shortened to one step: "A confirmation dialogue will appear, followed by a dialogue asking for your local administrator password. After confirming these steps, KA Lite will be uninstalled.".

File: docs/installguide/uninstall.rst [hunk 3]
```diff
@@ -11,14 +11,22 @@ _______
 Mac OSX
 _______
 
-1. Launch ``KA-Lite Monitor`` from your ``Applications`` folder.
+Option 1:
```
Comment (benjaoming) [inline docs/installguide/uninstall.rst]: Call it:

    Uninstallation from user interface
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

File: docs/installguide/uninstall.rst [hunk 4]
```diff
@@ -11,14 +11,22 @@ _______
...
-5. You will be prompted that "This will reset app. Are you sure?", just click on ``OK`` button.
-6. Another dialog will appear asking your ``Password``, type your password then click on ``Ok`` button.
-7. Quit the ``KA-Lite Monitor`` app (do not click the ``Apply`` button!).
-8. Move the ``KA-Lite Monitor`` app to ``Trash``.
+4. Click the ``Uninstall KA Lite`` from the ``Preferences`` tab.
+5. if you want to delete your KA Lite data folder checked the ``Delete KA Lite data folder`` checkbox.
+6. You will be prompted that ``Are you sure that you want to uninstall the KA Lite application?``, just click on ``OK`` button.
+7. Another dialog will appear asking your ``Password``, type your password then click on ``Ok`` button.
+
+Option 2:
```
Comment (benjaoming) [inline docs/installguide/uninstall.rst]: Call it:

    Uninstallation from command line
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

File: docs/installguide/uninstall.rst [hunk 5]
```diff
@@ -11,14 +11,22 @@ _______
...
+
+1. Open Terminal.
+2. Type ``bash /Applications/KA-Lite/KA-Lite_Uninstall.tool`` in your Terminal and press Enter.
+3. It will confirm if you want to keep or delete your KA Lite data folder.
```
Comment (benjaoming) [inline docs/installguide/uninstall.rst]: "You will be prompted to choose to keep or delete your data folder"

**Comment by mrpau-eduard** [comment]
Hi @benjaoming I update the installation docs base on your suggestion can you take a look at it again?
Thank you for reviewing it :)

File: docs/installguide/uninstall.rst [hunk 6]
```diff
@@ -11,14 +11,23 @@ _______
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Launch ``KA-Lite`` from your ``Applications`` folder.
 2. Click on the app icon at the menu bar.
 3. Click on ``Preferences`` in the menu option.
-4. Click the ``Reset App`` from the ``Advanced`` tab.
-5. You will be prompted that "This will reset app. Are you sure?", just click on ``OK`` button.
+4. Click the ``Uninstall KA Lite`` from the ``Preferences`` tab.
+5. A confirmation dialogue will appear, followed by a dialogue asking for your local administrator password. After confirming these steps, KA Lite will be uninstalled.
 6. Another dialog will appear asking your ``Password``, type your password then click on ``Ok`` button.
```
Comment (benjaoming) [inline docs/installguide/uninstall.rst]: This step should be removed, too, it's described above!

File: docs/installguide/uninstall.rst [hunk 7]
```diff
@@ -37,7 +46,7 @@ __________________
 You can remove KA Lite (when installed from pip or source distribution) with
 ``pip uninstall ka-lite`` or ``pip uninstall ka-lite-static`` (static version).
 
-
+333
```
Comment (benjaoming) [inline docs/installguide/uninstall.rst]: Weird artifact?

File: docs/installguide/uninstall.rst (hunk 6 above)
Comment (mrpau-eduard) [inline docs/installguide/uninstall.rst]: Ohh, sorry I accidentally revert the line from my stash. Now removing it.

**Comment by mrpau-eduard** [comment]
Done removing the redundant step @benjaoming .

**Comment by benjaoming** [comment]
> Done removing the redundant step @benjaoming

You missed my last comment ;)

File: docs/installguide/uninstall.rst (hunk 7 above)
Comment (mrpau-eduard) [inline docs/installguide/uninstall.rst]: Removing it now I didn't notice it sorry. Done removing it @benjaoming

**Comment by benjaoming** [comment]
Cool - @mrpau-eduard could you compress everything into a single commit?

Typically, you reset your feature branch onto the branch you wanna merge to.

```
git reset origin/0.17.x
git add .
git commit
```

**Comment by benjaoming** [comment]
@mrpau-eduard I've edited my above response, was putting some incomplete steps, better to just read the guide..

**Pull request closed without merging**